python-sdk/tests/sql/test_table.py

```diff
@@ -210,9 +210,9 @@ def test_if_table_object_can_be_pickled():
             "snowflake://astro-sdk",
         ),
         (
-            Connection(conn_id="test_conn", conn_type="sqlite", host="/tmp/sqlite.db"),
-            "/tmp/sqlite.db.test_tb",
-            "/tmp/sqlite.db",
+            Connection(conn_id="test_conn", conn_type="sqlite", host="tmp/sqlite.db"),
+            "tmp/sqlite.db.test_tb",
+            "sqlite://tmp/sqlite.db",
```

Comment:
Two comments here: 
- The namespace should not repeat the dataset name ("tmp/sqlite.db.test_tb").
- The tricky aspect of those local datasets that are private to the machine they are running on is we want the dataset ns+name to be unique. Otherwise if you run this on two different machines the lineage graph will assume that you are writing and reading the same dataset, which is not true. You need to make the namespace more unique by (for example) adding the machine name/IP in it.

Reply:
I have added the PR for the change https://github.com/astronomer/astro-sdk/pull/1178. Please review